Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -494,3 +498,9 @@ RetainPtr<NSError> nsErrorFromExceptionDetails(const WebCore::ExceptionDetails&)
 - (void)_requestTextExtractionForSwift:(WKTextExtractionRequest *)context;
 - (void)_requestTextExtraction:(CGRect)rect completionHandler:(void(^)(WKTextExtractionItem *))completionHandler;
 @end
+
+#endif // __cplusplus
+
+@interface WKWebView (NonCpp)
+
+@end
```

Comment:
Do we really want to re-declare ObjC interfaces from above? I would definitely prefer a more fine-grained use of `#ifdef __cplusplus` if you think that is possible.

Reply:
I originally had the same thought, but it seems like well over half of the file relies on C++ stuff, so I figured this was easier to follow than adding a ton of `#ifdef __cplusplus` everywhere. I can change it though if you want!